Automattic/sensei
# Footer style improvements for quiz on learning mode

Resolves https://github.com/Automattic/sensei/issues/7073

## Proposed Changes
* Fixed quiz actions block not adding block attributes during render
* Fixed Button spacings
* Updated styles for desktop and mobile as per the new design

## Testing Instructions


1. Check out this branch of Course theme https://github.com/Automattic/themes/pull/7317
2. Activate course theme
3. Create a course
4. Create a lesson in it with a quiz
5. Enable learning mode
6. Take the course as a student, go to the quiz page
7. Make sure the footer looks as expected
8. Check in both mobile and desktop mode
9. Now disable learning mode
10. Make sure it didn't break anything
11. Now repeat step 5-8 for all variations of course theme, top sensei themes, another block theme

Desktop:
<img width="1050" alt="image" src="https://github.com/Automattic/sensei/assets/6820724/907be25d-d8a5-4b43-80e4-7dfb243d119b">

Mobile:
<img width="491" alt="image" src="https://github.com/Automattic/sensei/assets/6820724/e848aa5e-924c-4222-b309-6dfc66787aad">

## Pre-Merge Checklist
